blog/2022-05-10-phase2.mdx

```diff
@@ -0,0 +1,73 @@
+
+The [Phase 1 leaderboard](https://phase1.testnet.ironfish.network/leaderboard) and [explorer](https://phase1.explorer.ironfish.network) are saved for historic/archival reasons.
+
+Phase 1 token distribution will happen at the same time as Phase 2 token distribution and closer to mainnet. When mainnet? When all the engineering tasks on our [roadmap](https://ironfish.network/roadmap) are complete. If you want this to go faster, consider applying on our [careers page](https://ironfish.network/careers/) for a full time engineering job at Iron FIsh :) or help us with open source contribution as part of the incentivized testnet program.
+What’s new in Phase 2?
+Phase 2 is meant to stress-test Iron Fish as well as incorporate some of the feedback we’ve heard from the community about Phase 1. We heard your feedback, and we’re making some changes!
+
+Phase 2 will have 4 categories (described in even more detail on the [testnet site](https://testnet.ironfish.network/about)):
+
+### Hosting a node
+
+Running an Iron Fish full node with telemetry on and a registered graffiti set gets you max of 10 points for every 12 consecutive hours of hosting (even if running multiple nodes). Points will be automatically assigned to you on the leaderboard. To make sure your node is properly set up to earn points, we made a custom command `ironfish testnet`. Go ahead and run that command before starting your node to make sure you’re all set.
+
+### Sending a transaction
+
+Sending a transaction of at least 0.1 coins to the Iron Bank account gets you 1 point. The public address for the Iron Fish bank is 5cfbcbf6e6f84e2bab6c17309f39908d7cdb8ae5ad4ab600bbb9101403a1edc2d03f71754764e8c66afc08 which is a mouthful, so we made a custom command `ironfish deposit` that constructs a default transaction of 0.1 to the Iron Bank bank for you.
+
+Note that while we’ve removed earning points for mining a block from Phase 2, the only way to get coins is by mining them. It is now possible (and easy!) to create and join pools. Feel free to [join the Iron Fish team pool](https://ironfish.network/docs/onboarding/miner-iron-fish#join-a-mining-pool) if you can’t find others. The faucet is disabled for this phase.
+
+### Reporting a bug
+
+If you see a bug, report it on our github issues page and submit the form to get points! One quality bug found is 100 points.
+
+### Open Source Contributions (Submitting a PR)
+
+Iron Fish is in heavy development. Not only are we focused on the bigger roadmap items for engineerings, but we’re constantly working on improving user experience. Contributing to the ironfish repo is immensely appreciated as it directly impacts the project getting to mainnet faster.
+
+We appreciate open source contributions so much that we are allocating a <b>_separate pool of coins_</b> just for PR contributions. Open source developers no longer have to compete with miners for points in the same pool.
```

Comment:
Minor but I think you can do `*_separate pool of coins_*` to get italic + bold

Reply:
fixed